sof_wrapper/api/fhir.py

```diff
@@ -13,7 +13,7 @@ def collate_results(*result_sets):
     results = {'resourceType': 'Bundle', 'entry': []}
 
     for rs in result_sets:
-        results['entry'].extend(rs['entry'])
+        results['entry'].extend(rs.get('entry', []))
```

Comment:
this will potentially create a number of empty list elements within `results['entry']` - would it not be better to skip over extending `results` if `rs.entry` isn't well defined?

Reply:
Good point- fixed